flask_user/forms.py

```diff
@@ -210,7 +221,7 @@ def validate(self):
             user, user_email = user_manager.find_user_by_email(self.email.data)
 
         # Handle successful authentication
-        if user and user.password and user_manager.verify_password(self.password.data, user):
+        if user and user_manager.verify_password(self.password.data, user):
```

Comment:
@pbugni I see now that you added that check. I'm confused why you needed it. With my change (and the way it was before), you can always check `user.password` within the body of `user_manager.verify_password` if you override that method on the `UserManager`.